### PR TITLE
compatibility with perl 5.25/5.26

### PR DESCRIPTION
**Pull request by chorny**

### CODE REVIEW

**Comment by chorny** [comment]
fixes issue #222

File: lib/DDG/Rewrite.pm [hunk 1]
```diff
@@ -159,9 +159,9 @@ sub _build_nginx_conf {
 		# the sorted keys into a single string.
 		# e.g. post_body '{"method":"$2","query":"$1","cleaned_query":"$1"}'
 		# Would give a $cache_keys value of '$1$2'
-		my $cache_keys = join '', sort keys {
+		my $cache_keys = join '', sort keys %{ {
```
Comment (chorny) [inline lib/DDG/Rewrite.pm]: 'keys' works only on hashes or arrays, but not on lists

Comment (GuiltyDolphin) [inline lib/DDG/Rewrite.pm]: @chorny By the way, what is the error raised when using the origininal version of this?

Comment (chorny) [inline lib/DDG/Rewrite.pm]: "Experimental keys on scalar is now forbidden".  See https://github.com/duckduckgo/duckduckgo/issues/222

**Comment by zachthompson** [comment]
Thank you.

**Comment by moollaza** [comment]
Looks like we missed another occurrence of this above. I've fixed it in #233 👍

**Pull request merged**